app/services/preserved_object_handler.rb

```diff
@@ -134,8 +182,10 @@ def logger_severity_level(result_code)
     when ARG_VERSION_LESS_THAN_DB_OBJECT then Logger::ERROR
     when UPDATED_DB_OBJECT then Logger::INFO
     when UPDATED_DB_OBJECT_TIMESTAMP_ONLY then Logger::INFO
-    when CREATED_NEW_OBJECT then Logger::WARN
+    when CREATED_NEW_OBJECT then Logger::INFO
```

Comment:
b/c the spec changed?  or should this line be dependent on `expect_to_create`?

Reply:
IIRC we changed it to INFO because when the object is created, we don't want to log that as a warning as the create method was called on purpose....Then if the method was accidentally called on an object thats not in the db  we log an error in the audit check? @jmartin-sul

Reply:
ah - maybe you're right about the spec changing.  I'm kinda at the "let's do it, fire it up, and let the users see if they like it" stage.

Reply:
yup, agreed w/ @SaravShah, i think by splitting `update_or_create` into two methods, we're pushing the responsibility for warning about an unexpected create out to the caller, which decides whether to create if the object isn't already there.

(which, from my point of view, is less an overall spec change, and more an implementation/decomposition change, which is to say we should still be logging a warning, which i think we're doing, but now it's here, AFAICT:  https://github.com/sul-dlss/preservation_core_catalog/pull/190/files#diff-7d6cab3132273db5a784afb09ac3b5e0R14)